js/script.js

```diff
@@ -313,21 +313,47 @@ function filmOutput(films) {
     films = films.sort(() => Math.random() - 0.5)
     //Pick 5 films
     for(let i=0; i<=4; i++) {
+        //Initialise film being added
         let film = films[i]
-        const leftFilm = document.createElement("section")
+
+        //Create div for posterimage to be added to
+        const posterBox = document.createElement("article")
+        posterBox.className="poster-box"
+        //Add the film poster to image tag
+        const poster = document.createElement("img")
+        poster.src = baseImageURL + "w500" + film.poster_path
+        poster.alt = "Poster for " + film.title
+        poster.className = "poster";
+        //Append the image to the div made for the poster
+        posterBox.append(poster);
+        posterBox.tabIndex = 0
+        //Append to the rest of the posters on the page
+        results.append(posterBox)
+
+        //Setup overlay
+        const filmOverlay = document.createElement("div");
+        filmOverlay.classList.add("film-overlay", "center", "width-lg");
+        //Add the x to close the overlay
+        const span = document.createElement("span");
+        span.className = "close"
+        span.innerHTML = `&times;`
+        //Add the box for left side of the overlay
+        const leftFilm  = document.createElement("section")

```

Comment:
adds to posterbox still, but also initialises a new posterbox inside, then appends to the whole results as a whole.

Will make it much easier to now alter the number of films outputted to a user, for their choice of films relates #25